src/components/map/BiodivInformation.vue

```diff
@@ -14,15 +14,16 @@
       <span>{{ months.join(", ") }}</span>
     </p>
 
-    <!-- TODO : find a light way to securely show this content -->
-    <p>{{ description }}</p>
+    <!-- content formatting done with "he" html entities decoder -->
+    <p v-for="descPar of descriptionParagraphs">{{ descPar }}</p>
     <a :href="data.infoUrl" target="_blank" rel="noopener" v-translate>More info</a> |
     <a :href="data.kmlUrl" target="_blank" rel="noopener">KML</a>
   </modal-window>
 </template>
 
 <script>
 
+  import he from 'he';
```

Comment:
I'm surprised to see this lib, we removed from our stack 6 month ago as it's very heavy. It works on your end???

For removing HTML entities , we have a function in utils : `utils.decodeHtmlEntities()` it also removes html markup, so you can skip discard all remaining html tags.